### PR TITLE
Add Recent Reactions to the Admin Users' Page

### PR DESCRIPTION
**Pull request by juliannatetreault**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
It looks like this code got mixed up in a rebase I did and was removed from the Admin UI. 😞 This PR adds the "Recent Reactions" section _back_ to the Admin Users' UI (`/admin/users/:id/edit`). 

Additionally, this PR: 
- Adds all related logic that makes the negative reaction reports render correctly (`#set_related_reactions` in the `Admin::UsersController`)
- Adjusts the JS in `_reports.html.erb` to target the `#reports-row` rather than the `#reactions-row`
- Adds tests for both "Recent Reactions" and "Recent Reports" to `/spec/reqeusts/admin/users_spec.rb`

## Related Tickets & Documents
N/A

## QA Instructions, Screenshots, Recordings
- To QA these changes, please ensure that you have an `admin` role and navigate to `/admin/users/:id/edit`. Ensure that the "Recent Reactions" section exists above the "Recent Reports" section:
Screen Shot 2020-10-06 at 1 45 18 PM

- Toggle the "Recent Reactions" section to see a list of all of the vomit reactions associated with a user. If the user does not have any related vomit reactions, you should see a message saying "_Nothing negative to see here! 👀_ " instead:
Screen Shot 2020-10-06 at 1 47 43 PM
Screen Shot 2020-10-06 at 1 47 59 PM

## Added tests?

- [x] yes
- [ ] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.forem.com
- [ ] readme
- [x] no documentation needed

## [optional] Are there any post-deployment tasks we need to perform?
N/A

## [optional] What gif best describes this PR or how it makes you feel?

Michelle Obama Facepalming

### CODE REVIEW

**Review by Zhao-Andy — APPROVED** [review]
LGTM! Maybe a quick request spec to test that the page shows the recent reactions?

**Comment by juliannatetreault** [comment]
> LGTM! Maybe a quick request spec to test that the page shows the recent reactions?

Will do! Working on it now, @Zhao-Andy. 😊

File: spec/requests/admin/users_spec.rb [hunk 1]
```diff
@@ -40,6 +40,38 @@
       sign_out(admin)
       expect { get "/admin/users/#{user.id}/edit" }.to raise_error(Pundit::NotAuthorizedError)
     end
+
+    it "displays the 'Recent Reactions' section" do
```
Comment (rhymes) [inline spec/requests/admin/users_spec.rb:44]: should we call that section "Recent negative reactions?"

**Review by rhymes — APPROVED** [review]
Great catch! I wonder if it should be called "recent negative reactions" but I guess there's a reason for the name to be generic

**Pull request merged**